src/test/e2e/api/admin/event.e2e.test.ts

```diff
@@ -22,6 +24,7 @@ beforeAll(async () => {
         },
     });
     eventStore = db.stores.eventStore;
+    eventService = new EventService(db.stores, { getLogger });
```

Comment:
Maybe it's safe to remove `eventStore` variable above

Reply:
Went with [this](https://github.com/Unleash/unleash/pull/4830/commits/9266c7aaf1d9279515c6f9ddce3ece9fd73daadc) approach. I don't think the `deleteAll` method should be exposed on the service layer, but if you disagree I can add this proxy.

Reply:
I agree! And I like your approach :+1: